conn.go

```diff
@@ -600,7 +603,8 @@ func (c *Conn) recvV5Frame(ctx context.Context) error {
 	}
 
 	if isSelfContained {
-		// TODO handle case when there are more than 1 envelop inside the frame
+		// Handle case when there is more than one envelope inside the frame
+		// TODO: Implement handling for multiple envelopes
```

Comment:
It can be deleted because handling of multi-framed envelopes already implemented below